requirements/libraries.txt

```diff
@@ -10,6 +10,7 @@ django-extensions==2.1.0
 django-flags==3.0.0
 django-haystack==2.7.0
 django-localflavor==1.1
+django-mptt==0.9.0
```

Comment:
Would you mind adding a comment here indicating that this is required by the teachers-digital-platform package?